packages/react/cypress/component/auto/form/AutoForm.cy.tsx

```diff
@@ -32,6 +32,14 @@ describeForEachAutoAdapter("AutoForm", ({ name, adapter: { AutoForm }, wrapper }
     cy.contains("Saved Widget successfully");
   });
 
+  // TODO: fix the issue of sending a GraphQL mutation request that the hasMany field value is null.
```

Comment:
I think @pistachiobaby may have fixed this now?

Reply:
It's not fixed at the GraphQL level (related ticket: GGT-6605). The "has many" field input component for MUI is not done, so I can temporarily exclude the `rounds` field first which is a `hasMany` type.

Reply:
I just added a test on the gadget side to prove that
https://github.com/gadget-inc/gadget/pull/12053